M2/Macaulay2/m2/methods.m2

```diff
@@ -656,7 +658,7 @@ runHooks(Symbol,                  Thing) := true >> opts -> (key,        args) -
 runHooks(Sequence,                Thing) := true >> opts -> (key,        args) -> runHooks(getHookStore(key, false), key, args, opts)
 runHooks(MutableHashTable, Thing, Thing) := true >> opts -> (store, key, args) -> (
     store = if store#?key then store#key else (
-	if debugLevel > 1 then printerr("runHooks: no hooks installed for ", toString key); return );
+	if debugHooksLevel > 1 then printerr("runHooks: no hooks installed for ", toString key); return );
```

Comment:
Should `debugHooksLevel` be documented anywhere?  Should it be exported?

Right now we *kind of* document `debugLevel` for hooks.  It's not mentioned explicitly, but it's set in one of the examples at [using hooks](https://macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_using_sphooks.html).

Reply:
I don't think it's worth exporting. Neither is `HookAlgorithms`, `HookPriority`, etc.